Downloads/ClearviewMemorials/wp-content/plugins/hybrid-composer/index.php

```diff
@@ -654,31 +654,34 @@ function hc_get_pages_list($selected_id = "") {
 </select>
 <?php
 }
-function hc_json($content) { 
-    return str_replace(array('\"', '\\\\',"\'"), array('"', '\\',"'"), $content); 
-} 
-
-//AJAX
-
-function hc_ajax_update_editor_mode() {
-    die(update_post_meta($_POST['id'], 'hc-editor-mode', $_POST['value']));
-}
+function hc_json($content) {
```

Comment:
@weasek updating this function is what fixes the site and allows it to load in php8.x